tools/protovalidate-conformance/internal/cases/cases_any.go

```diff
@@ -117,5 +118,21 @@ func anySuite() suites.Suite {
 				},
 			),
 		},
+		"compilation/wrong_type/scalar": {
+			Message:  &cases.AnyWrongTypeScalar{Val: "test"},
```

Comment:
The compilation error should occur even when the field isn't populated, and isn't validated, right?

Then I wouldn't populate the field in the new cases:

```suggestion
			Message:  &cases.AnyWrongTypeScalar{},
```